libnavigation-base/src/main/java/com/mapbox/navigation/base/extensions/RouteOptionsEx.kt

```diff
@@ -32,7 +38,45 @@ fun RouteOptions?.supportsRouteRefresh(): Boolean {
             it == DirectionsCriteria.ANNOTATION_MAXSPEED ||
             it == DirectionsCriteria.ANNOTATION_SPEED ||
             it == DirectionsCriteria.ANNOTATION_DURATION ||
-            it == DirectionsCriteria.ANNOTATION_DISTANCE
+            it == DirectionsCriteria.ANNOTATION_DISTANCE ||
+            it == DirectionsCriteria.ANNOTATION_CLOSURE
     } ?: false
     return isTrafficProfile && isOverviewFull && hasCongestionOrMaxSpeed
 }
+
+/**
+ * Applies the [RouteOptions] that are required for the route request to execute
+ * or otherwise recommended for the Navigation SDK and all of its features to provide the best car navigation experience.
+ */
+fun RouteOptions.Builder.applyDefaultNavigationOptions(): RouteOptions.Builder = apply {
```

Comment:
This https://github.com/mapbox/mapbox-navigation-android/pull/4320#discussion_r626616826 still applies. What do you think?

Reply:
My thoughts are in https://github.com/mapbox/mapbox-navigation-android/pull/4320#issuecomment-839710568. I hope that we can be transparent about it and document it properly instead of changing values under the hood and introducing surprising behavior.

Reply:
Ohh I meant only for the required options.